### PR TITLE
Added Pod security deprecation note

### PR DESCRIPTION
**Pull request by Kartik494**
Fixes: #31998

### CODE REVIEW

File: content/en/docs/concepts/storage/ephemeral-volumes.md [hunk 1]
```diff
@@ -128,6 +128,10 @@ instructions.
 As a cluster administrator, you can use a [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/) to control which CSI drivers can be used in a Pod, specified with the
 [`allowedCSIDrivers` field](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podsecuritypolicyspec-v1beta1-policy).
 
+{{< caution >}}
+PodSecurityPolicy is deprecated as of Kubernetes v1.21, and will be removed in v1.25
+{{< /caution >}}
```
Comment (sftim) [inline content/en/docs/concepts/storage/ephemeral-volumes.md:133]:
```suggestion
{{< feature-state for_k8s_version="v1.21" state="deprecated" >}}

As a cluster administrator, you can use a [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/) to control which CSI drivers can be used in a Pod, specified with the
[`allowedCSIDrivers` field](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podsecuritypolicyspec-v1beta1-policy).

{{< note >}}
PodSecurityPolicy is deprecated and will be removed in the Kubernetes v1.25 release.
{{< /note >}}
```

**Review by sftim — COMMENTED** [review]
/lgtm

Here's how I'd do that update @Kartik494

File: content/en/docs/concepts/storage/ephemeral-volumes.md (hunk 1 above)
Comment (Kartik494) [reply content/en/docs/concepts/storage/ephemeral-volumes.md]: @sftim addressed the changes PTAL!

**Comment by Kartik494** [comment]
@sftim PTAL!

File: content/en/docs/concepts/storage/ephemeral-volumes.md [hunk 2]
```diff
@@ -125,9 +125,15 @@ driver. These attributes are specific to each driver and not
 standardized. See the documentation of each CSI driver for further
 instructions.
```
Comment (sftim) [inline content/en/docs/concepts/storage/ephemeral-volumes.md:127]: One more suggestion
```suggestion

#### CSI driver restrictions

```

Comment (Kartik494) [reply content/en/docs/concepts/storage/ephemeral-volumes.md:127]: @sftim Addressed the changes!

**Comment by Kartik494** [comment]
@sftim Please let me know if any other changes are required on this, i have addressed the changes earlier.
Thanks!

**Pull request merged**